### PR TITLE
Implemented Ondu Spiritdancer, Nyxborn Behemoth and Onakke Oathkeeper

### PR DESCRIPTION
**Pull request by nshcat**
As the title says :)

### CODE REVIEW

File: forge-gui/res/cardsfolder/o/ondu_spiritdancer.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Name:Ondu Spiritdancer
+ManaCost:4 W
+Types:Creature Kor Cleric
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Enchantment.YouCtrl | TriggerZones$ Battlefield | ResolvedLimit$ 1 | OptionalDecider$ You | Execute$ TrigCopy | TriggerDescription$ Whenever an enchantment enters the battlefield under your control, you may create a token that’s a copy of it. Do this only once each turn.
+SVar:TrigCopy:DB$ CopyPermanent | Defined$ TriggeredCardLKICopy
+Oracle:Whenever an enchantment enters the battlefield under your control, you may create a token that’s a copy of it. Do this only once each turn.
```
Comment (Agetian) [inline forge-gui/res/cardsfolder/o/ondu_spiritdancer.txt:7]: DeckHas:Ability$Token
(+ there's an empty line at the end of the file, needs to be removed)

File: forge-gui/res/cardsfolder/o/onakke_oathkeeper.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Onakke Oathkeeper
+ManaCost:1 W
+Types:Creature Ogre Spirit
+PT:0/4
+S:Mode$ CantAttackUnless | ValidCard$ Creature | Target$ Planeswalker.YouCtrl | Cost$ 1 | Description$ Creatures can't attack planeswalkers you control unless their controller pays {1} for each creature they control that’s attacking a planeswalker you control.
+A:AB$ ChangeZone | Cost$ 4 W W ExileFromGrave<1/CARDNAME> | ActivationZone$ Graveyard | Origin$ Graveyard | Destination$ Battlefield | ValidTgts$ Planeswalker.YouOwn | SpellDescription$ Return target planeswalker card from your graveyard to the battlefield.
+Oracle:Creatures can’t attack planeswalkers you control unless their controller pays {1} for each creature they control that’s attacking a planeswalker you control.\n{4}{W}{W}, Exile Onakke Oathkeeper from your graveyard: Return target planeswalker card from your graveyard to the battlefield.
+
```
Comment (Agetian) [inline forge-gui/res/cardsfolder/o/onakke_oathkeeper.txt:8]: DeckHas:Ability$Graveyard
(+ an empty line at the end of the file - needs to be removed)

File: forge-gui/res/cardsfolder/o/onakke_oathkeeper.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Name:Onakke Oathkeeper
+ManaCost:1 W
+Types:Creature Ogre Spirit
+PT:0/4
+S:Mode$ CantAttackUnless | ValidCard$ Creature | Target$ Planeswalker.YouCtrl | Cost$ 1 | Description$ Creatures can't attack planeswalkers you control unless their controller pays {1} for each creature they control that’s attacking a planeswalker you control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/o/onakke_oathkeeper.txt:5]: noncurly versions please
’ = '

File: forge-gui/res/cardsfolder/o/onakke_oathkeeper.txt (hunk 2 above)
Comment (nshcat) [reply forge-gui/res/cardsfolder/o/onakke_oathkeeper.txt]: Thank you :)

File: forge-gui/res/cardsfolder/o/onakke_oathkeeper.txt (hunk 3 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/o/onakke_oathkeeper.txt:5]: look closely depending on your font, the apostrophes are not the same
in fact you already mixed them in that description
for consistency we only use the simple version (not sure if some older skins still behave better with ASCII or we can eventually do a big update)

Comment (nshcat) [reply forge-gui/res/cardsfolder/o/onakke_oathkeeper.txt]: Ah okay thank you, fixed :)

**Review by tool4ever — APPROVED** [review]

**Pull request merged**